ansible/roles/actors/tasks/main.yml

```diff
@@ -0,0 +1,25 @@
+---
+- name: Get leapp actors main repository
+  git:
+      repo: "{{ actors.repo }}"
+      dest: "{{ actors.dest }}"
+
+- name: Install packages requested by some actors, if necessary
+  become: true
+  become_method: sudo
```

Comment:
It is better to set `become_method` in ansible.cfg / playbook - tasks themselves shouldn't need to know how the privilege is being escalated